src/software/ai/hl/stp/tactic/dribble/simulated_dribble_tactic_test.cpp

```diff
@@ -0,0 +1,291 @@
+#include <gtest/gtest.h>
+
+#include <utility>
+
+#include "software/ai/hl/stp/tactic/dribble/dribble_tactic.h"
+#include "software/geom/algorithms/contains.h"
+#include "software/simulated_tests/simulated_tactic_test_fixture.h"
+#include "software/simulated_tests/terminating_validation_functions/ball_at_point_validation.h"
+#include "software/simulated_tests/terminating_validation_functions/robot_received_ball_validation.h"
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/test_util/test_util.h"
+#include "software/time/duration.h"
+#include "software/world/world.h"
+
+class SimulatedMoveTacticTest : public SimulatedTacticTestFixture
```

Comment:
Would be good to add a test case checking that we don't break the 1 meter dribble limit

Reply:
good point

Reply:
It doesn't seem like something that should be enforced at the tactic level. For example ball placement doesn't have a dribble limit

Reply:
implemented it with a flag to turn it off it